docs/setup/kubeedge_install_keadm.md

```diff
@@ -0,0 +1,219 @@
+# Setup from KubeEdge Installer
+
+Please refer to KubeEdge Installer proposal document for details on the motivation of having KubeEdge Installer.
+It also explains the functionality of the proposed commands.
+[KubeEdge Installer Doc](https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/keadm-scope.md/)
+
+## Limitation
+
+- Currently support of `KubeEdge installer` is available only for Ubuntu OS. CentOS support is in-progress.
+
+## Getting KubeEdge Installer
+
+There are currently two ways to get keadm
+
++ Download from [KubeEdge Release](<https://github.com/kubeedge/kubeedge/releases>)
+
+  1. Go to [KubeEdge Release](<https://github.com/kubeedge/kubeedge/releases>) page and download `keadm-$VERSION-$OS-$ARCH.tar.gz.`.
+  2. Untar it at desired location, by executing `tar -xvzf keadm-$VERSION-$OS-$ARCH.tar.gz`.
+  3. kubeedge folder is created after execution the command.
+
++ Building from source
+
+  1. Download the source code.
+  
+      ```
+      git clone https://github.com/kubeedge/kubeedge.git $GOPATH/src/github.com/ kubeedge/kubeedge
+      cd $GOPATH/src/github.com/kubeedge/kubeedge/keadm
+      make
+      ```
+  2. Binary `keadm` is available in current path.
+
+
+## Setup Cloud Side (KubeEdge Master Node)
+
+Referring to `KubeEdge Installer Doc`, the command to install KubeEdge cloud component (edge controller) and pre-requisites.
+Port 8080, 6443 and 10000 in your cloud component needs to be accessible for your edge nodes.
+
+keadm by default can install Docker, Kubernetes and KubeEdge. It also provide flag by which specific versions can be set.
+
+1. Execute `keadm init`
+
+    Command flags
+    The optional flags with this command are mentioned below
+
```

Comment:
we should mention, that keadm needs super user rights (or root rights) to run successfully.